scripts/aws_clear_instance.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo "Removing application and dependencies"
+
+if [ -d "/home/notify-app/notifications-api" ]; then
+    # Remove and re-create the directory
+    rm -rf /home/notify-app/notifications-api
+    mkdir -vp /home/notify-app/notifications-api
+    # Remove installed py3 packages
+    pip3 freeze | xargs pip3 uninstall -y
```

Comment:
I wouldn't worry about leaving some dependencies on the node, we plan to recycle them every week eventually.